mbedtls/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "mbedtls"
-version = "0.8.2"
+version = "0.8.3"
```

Comment:
You changed the interface, so these versions are not compatible with one another.
When you only update the patch version, and dependencies are defined as `mbedtls = 0.8`, Cargo may automatically pick up the new version. Unfortunately that will lead to compile issues.
https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html